reame.txt

```diff
@@ -0,0 +1,5 @@
+Głównym celem tego projektu było zapoznanie się ze rodzajami list, ich implementacją oraz ręcznym i automatycznym scrollowaniem.
```

Comment:
Incorrect file. Add `README.md` instead of `readme.txt`.

Reply:
Just change file type from txt to md without changing it's?